src/Tribe/Cost_Utils.php

```diff
@@ -208,7 +208,7 @@ protected function get_cost_by_func( $costs = null, $function = 'max' ) {
 		}
 
 		// use a regular expression instead of is_numeric
-		if ( ! preg_match( $this->get_cost_regex(), $cost ) ) {
+		if ( empty( $cost ) ) {
```

Comment:
In changing this line we should also update the comment above it (or remove it).